demos/deploySolution/index.html

```diff
@@ -264,43 +303,61 @@ <h3>Deploy a Solution</h3>
               // Upon a successful login, update the session with the new session.
               session = newSession;
               updateSessionInfo(session);
-              deploySolution(session);
+              deploySolution(solutionId, folderId, srcCreds, session, useExisting);
             }).catch(function (error) {
               console.log(error);
             });
           }
+        }
+
+        // Deploy!
+        function deploySolution(solutionId, folderId, srcCreds, destCreds, useExisting) {
+          const startTime = Date.now();
+          const createdItems = [];
 
-          // Deploy!
-          function deploySolution(_destCreds) {
-            main.deploySolution(
+          let deployPromise = Promise.resolve("<i>No Solution(s) provided</i>");
+          const progressFcn =
+            function (percentDone, jobId, progressEvent) {
+              if (progressEvent) {
+                createdItems.push(progressEvent.data)
```

Comment:
;

Reply:
Or am I slowly getting the idea that these are elements that can be *added* to demoCommon?

Reply:
very minor thing but was just seeing a few missing ; no big deal really